Dockerfile

```diff
@@ -64,11 +64,10 @@ COPY --from=builder /etc/ssl/certs /etc/ssl/certs
 COPY --from=builder workspace/bin/kcp-front-proxy workspace/bin/kcp workspace/bin/virtual-workspaces workspace/bin/cache-server /
 COPY --from=builder workspace/bin/kubectl-* /usr/local/bin/
 COPY --from=builder workspace/bin/kubectl /usr/local/bin/
+
 ENV KUBECONFIG=/etc/kcp/config/admin.kubeconfig
-# Use uid of nonroot user (65532) because kubernetes expects numeric user when applying pod security policies
-RUN ["/busybox/sh", "-c", "mkdir -p /data && chown 65532:65532 /data"]
 USER 65532:65532
-WORKDIR /data
-VOLUME /data
+WORKDIR /etc/kcp
```

Comment:
Does the `WORKDIR` matter? Otherwise I'd just remove it.
```suggestion
```

Reply:
Good point.